statham/schema/elements/meta.py

```diff
@@ -102,6 +103,7 @@ def __new__(
         cls.const = get_value(const, "const")
         cls.enum = get_value(enum, "enum")
         cls.required = get_value(required, "required")
+        cls.description = get_value(description, "description")
```

Comment:
Okay so it can be achieved with `__init_subclass__`:

```diff
diff --git a/statham/schema/elements/object.py b/statham/schema/elements/object.py
index 4d9db16..fa5b3a2 100644
--- a/statham/schema/elements/object.py
+++ b/statham/schema/elements/object.py
@@ -35,6 +35,11 @@ class Object(metaclass=ObjectMeta):
     properties: ClassVar[Dict[str, _Property]]
     default: ClassVar[Any]
     additionalProperties: ClassVar[Union[Element, bool]]
+    description: ClassVar[str]
+
+    def __init_subclass__(cls, *args, **kwargs):
+        if cls.__doc__ and cls.description is NotPassed():
+            cls.description = cls.__doc__
 
     def __new__(
         cls, value: Any = NotPassed(), property_: _Property = UNBOUND_PROPERTY
diff --git a/tests/schema/elements/test_object.py b/tests/schema/elements/test_object.py
index 177316f..8551009 100644
--- a/tests/schema/elements/test_object.py
+++ b/tests/schema/elements/test_object.py
@@ -497,3 +497,26 @@ def test_object_property_override():
 
     with no_raise():
         _ = ChildObject(dict(value="a"))
+
+
+class TestObjectDescription:
+    @staticmethod
+    def test_that_description_is_read_from_kwarg():
+        class MyObject(Object, description="My description"):
+            pass
+
+        assert MyObject.description == "My description"
+
+    @staticmethod
+    def test_that_description_is_read_from_docstring():
+        class MyObject(Object):
+            """My docstring."""
+
+        assert MyObject.description == "My docstring."
+
+    @staticmethod
+    def test_that_description_kwargs_takes_precedence():
+        class MyObject(Object, description="My description"):
+            """My docstring."""
+
+        assert MyObject.description == "My description"

```

Reply:
Awesome, I will include that later today or tomorrow.